src/routes/index.js

```diff
@@ -4,32 +4,11 @@ const Home = require('./home');
 const CounterRoute = require('./counter');
 
 // Create routes
-module.exports = (store) => ({
+module.exports = (store) => ([{
     path: '/',
     component: CoreLayout,
-    indexRoute: Home,
     childRoutes: [
+        Home,
         CounterRoute(store)
     ]
```

Comment:
No more `indexRoute`! I always thought that was weird anyway. Using `exact: true` effectively replaces that prop